tspng/implantation.py

```diff
@@ -0,0 +1,74 @@
+# import statements
+import os
+
+from PIL import Image
+from PIL.PngImagePlugin import PngInfo
+from tspng import MIME_TYPE
+from typing import Dict
+
+
+def _implant_data(file: str, image: str, mime_type: str = MIME_TYPE) -> Dict:
```

Comment:
Eventually, the `file` argument should be changed to `data: str` and it should be a valid JSON string. Then, the `implant_to_file` renamed to `implement_from_file`, which would read the file, validate as JSON, and then pass to this function as the `data: str`.